docs/src/remap/remap.adoc

```diff
@@ -384,13 +384,13 @@ all of them make sense. The following combinations are useful idioms:
   internal functions may be accessed from the prolog and epilog
   handlers. Also, a longer rope to hang yourselves.
 
-`python=`'<pythonfunction>' `modalgroup=`'<group>'::
+``python=``_<pythonfunction>_ ``modalgroup=``_<group>_::
   Directly call to a Python function without any argument conversion.
   The most powerful way of remapping a code and going straight to
   Python. Use this if you don't need an NGC procedure, or NGC is
   just getting in your way.
 
-`argspec=`'<words>' `python=`'<pythonfunction>' `modalgroup=`'<group>'::
+``argspec=``_<words>_ ``python=``_<pythonfunction>_ ``modalgroup=``_<group>_::
```

Comment:
and here

Reply:
should all be fine now in the latest upload